omegaconf/container.py

```diff
@@ -296,7 +296,7 @@ def pretty(self, resolve=False):
         :return: A string containing the yaml representation.
         """
         container = OmegaConf.to_container(self, resolve=resolve, enum_to_str=True)
-        return yaml.dump(container, default_flow_style=False)
+        return yaml.dump(container, default_flow_style=False, allow_unicode=allow_unicode)
```

Comment:
I think it's safe to just allow unicode.
no need to extend the interface for this.

Reply:
Removed the interface extension.